src/main/java/org/jenkinsci/plugins/ghprb/extensions/build/GhprbCancelBuildsOnUpdate.java

```diff
@@ -39,15 +39,15 @@ public Boolean getOverrideGlobal() {
         return overrideGlobal == null ? false : overrideGlobal;
     }
 
-    private void cancelCurrentBuilds(AbstractProject<?, ?> project,
+    private void cancelCurrentBuilds(Job<?, ?> project,
                                      Integer prId) {
         if (getOverrideGlobal()) {
             return;
         }
         
         logger.log(Level.FINER, "New build scheduled for " + project.getName() + " on PR # " + prId + ", checking for queued items to cancel.");
         Queue queue = Jenkins.getInstance().getQueue();
-        for (Queue.Item item : queue.getItems(project)) {
+        for (Queue.Item item : queue.getItems()) {
```

Comment:
Doesn't this mean that you'd cancel builds of projects that aren't about to be rescheduled?  Is that the desired behavior?